packages/postcss-merge-longhand/src/lib/isCustomProp.js

```diff
@@ -1 +1,3 @@
-export default (node) => ~node.value.search(/var\s*\(\s*--/i);
+export default (node) =>
+  ~node.value.search(/var\s*\(\s*--/i) ||
+  ~node.value.search(/(constant|env)\s*\(/i);
```

Comment:
Let's rewrite this to `node.value.search(/(var|constant|env)\(/i)`, no need check on `--`

Reply:
Fixed.